src/core/Akka.Streams.Tests/Dsl/FlowSpec.cs

```diff
@@ -675,7 +675,7 @@ protected override bool AroundReceive(Receive receive, object message)
             Func<Flow<TOut, TOut2, Unit>> createGraph = () =>
             {
                 var stage = new FaultyFlowStage<TOut, TOut2>();
-                var assembly = new GraphAssembly(new IGraphStageWithMaterializedValue[] { stage }, new[] { Attributes.None },
+                var assembly = new GraphAssembly(new IGraphStageWithMaterializedValue<Shape, object>[] { stage }, new[] { Attributes.None },
```

Comment:
From what I can see in the usage of `IGraphStageWithMaterializedValue<Shape, object>`, the second generic parameter is always object. Feeling like something is wrong here. Either we should remove it and make object explicitly, or track this deep down to the point when object is cast to destination type.